pulp_rpm/app/models/repository.py

```diff
@@ -51,6 +51,12 @@ class RpmRemote(Remote):
     TYPE = "rpm"
     sles_auth_token = models.CharField(max_length=512, null=True)
 
+    uln_user = models.CharField(max_length=512, null=True)
+
+    uln_pwd = models.CharField(max_length=512, null=True)
+
+    uln_server_base_url = models.CharField(max_length=512, null=True)
```

Comment:
Since we have a dedicated Remote, we can remove those ones from the RpmRemote I believe.

Reply:
Done.